test/Test_RealComponents.ml

```diff
@@ -0,0 +1,188 @@
+open Base
+
+open LoopInvGen
+
+let value_of : Value.t -> float =
+  function [@warning "-8"]
+  | Real x -> x
+
+let add_eval = (List.find_exn RealComponents.translation
```

Comment:
Great to have these tests for individual components.  Have you also tested that PIE works with your new type?  I.e. can you do precondition inference where the precondition involves reals?  If so, it'd be nice to add some tests for that capability.  Do we already have existing tests for PIE somewhere?  Then you could add more there.

Reply:
Yes I have added some jobs in App.ml that use reals and they returned correct preconditions. I didn't add them here since my App.ml at this point has a pretty large amount of jobs and I didn't think you would want those in your version. Is there another way I should be testing PIE with reals?

Reply:
There is also `Test_PIE.ml` which sort of does integration testing for the precondition inference routines. You could add a job for Reals over there. May be just copy your App.ml job :)

Reply:
Yes, it'd be great to add a few PIE tests there, Julia.  Thanks.